doc/sphinx/source/recipes/recipe_hydrology.rst

```diff
@@ -35,6 +40,35 @@ User settings in recipe
    * start_year: 1979
    * end_year: 1979
 
+
+
+#. recipe_marrmot.yml
+
+   *Required preprocessor settings:*
+
+        *extract_shape:*
+
+            * shapefile: meuse_hydrosheds.shp (MARRMoT is a hydrological Lumped model that needs catchment-aggregated forcing data. The catchment is provided as a shapefile in Auxiliary folder in config-user.yml. The name of the shapefile is set in the recipe, here. For example, meuse_hydrosheds.shp).
+            * method: contains
```

Comment:
It would be good to mention that this setting should not be changed, also for `crop`

Reply:
Done.